src/css/styles/_general.scss

```diff
@@ -0,0 +1,134 @@
+@font-face {
+  font-display: swap;
+  font-family: "Praktika Bold";
+  src: url("fonts/praktika-boldextended-webfont.eot");
+  src:
+    url("fonts/praktika-boldextended-webfont.eot?#iefix") format("embedded-opentype"),
+    url("fonts/praktika-boldextended-webfont.woff2") format("woff2"),
+    url("fonts/praktika-boldextended-webfont.woff") format("woff"),
+    url("fonts/praktika-boldextended-webfont.ttf") format("truetype"),
+    url("fonts/praktika-boldextended-webfont.svg#praktikabold_extended") format("svg");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-display: swap;
+  font-family: "Praktika Medium";
+  src: url("fonts/praktika-medium-webfont.eot");
+  src:
+    url("fonts/praktika-medium-webfont.eot?#iefix") format("embedded-opentype"),
+    url("fonts/praktika-medium-webfont.woff2") format("woff2"),
+    url("fonts/praktika-medium-webfont.woff") format("woff"),
+    url("fonts/praktika-medium-webfont.ttf") format("truetype"),
+    url("fonts/praktika-medium-webfont.svg#praktikamedium") format("svg");
+  font-style: normal;
+  font-weight: normal;
+}
+
+.app {
+  background-color: #fff;
+}
+
+.nav-link,
+.banner-header {
+  font-family: "Praktika Bold", "Source Sans Pro", "Lucida Grande", sans-serif;
+}
+
+.app,
+.nav-link--mobile-kontakt {
+  font-family: "Praktika Medium", "Source Sans Pro", "Lucida Grande", sans-serif;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  max-width: 1600px;
+  padding: 0;
+  overflow: hidden;
+}
+
+.h-white {
+  color: #fff !important;
+}
+
+.h-black {
+  color: #000 !important;
+}
+
+.h-dark {
+  color: $col-txt-gray !important;
+}
+
+.h-light {
+  color: $col-txt-gray-light !important;
+}
+
+.h-orange {
+  color: $col-orange !important;
+}
+
+.dropdown-img {
+  padding-bottom: 0.2em;
+  padding-left: 0.2em;
+}
+
+.h-border-bottom {
+  border-bottom: solid 4px;
+}
+
+.h-border-bottom--orange {
+  border-bottom: solid $col-orange 4px;
+}
+
+.h-border-bottom--thin-gray {
+  border-bottom: solid $col-banner-gray-dark 2px;
+}
+
+.h-h1 {
+  margin-bottom: 3rem;
+  font-size: 3.5rem;
+  line-height: 110%;
+}
+
+.h-txt24 {
+  font-size: 1.6rem;
+}
+
+.h-txt16 {
+  font-size: 1rem;
+}
+
+.h-txt10 {
+  font-size: 0.8rem;
+}
+
+.h-txt9 {
+  font-size: 0.6rem;
+}
+
+@media (max-width: 991px) {
+  .h-h1 {
+    font-size: 2.8rem;
+  }
+}
+
+@media (max-width: 575.98px) {
+  .h-h1 {
+    font-size: 1.6rem;
+  }
+
+  .h-txt24 {
+    font-size: 1rem;
+  }
+
+  .h-txt10 {
+    font-size: 0.6rem;
+  }
+}
```

Comment:
Please move all classes connected with text to `_typography.scss`
- [ ] `_typography.scss`